scripts/qa_crowbarsetup.sh

```diff
@@ -4151,7 +4155,6 @@ function oncontroller_testsetup
     wait_image_active "$imageid" testsetup
 
     if [[ $want_ldap = 1 ]] ; then
-        install_suse_ca
```

Comment:
This one might have been insufficient anyway, because keystone might have been running on another (HA) controller node.